<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDocumentDialog</class>
 <widget class="QDialog" name="AddDocumentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QWidget" name="dialogContainer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>500</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#dialogContainer {
	border-radius: 12px;
	background-color: white;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>18</number>
      </property>
      <item>
       <widget class="QWidget" name="titleBarWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>18</number>
         </property>
         <property name="leftMargin">
          <number>18</number>
         </property>
         <property name="topMargin">
          <number>16</number>
         </property>
         <property name="rightMargin">
          <number>18</number>
         </property>
         <property name="bottomMargin">
          <number>16</number>
         </property>
         <item>
          <widget class="QLabel" name="dialogTitle">
           <property name="styleSheet">
            <string notr="true">color: #1A1A1A;
font-family: Open Sans;
font-weight: 500;
font-size: 22px;
line-height: 30px;
letter-spacing: 0%;</string>
           </property>
           <property name="text">
            <string>Add Document</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="closeButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../../Resources/Resources.qrc">
             <normaloff>:/images/Project/cross-icon.svg</normaloff>:/images/Project/cross-icon.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="contentWidget" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>24</number>
         </property>
         <property name="leftMargin">
          <number>24</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>24</number>
         </property>
         <property name="bottomMargin">
          <number>24</number>
         </property>
         <item>
          <widget class="QWidget" name="documentNameWidget" native="true">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QVBoxLayout" name="documentNameLayout">
            <property name="spacing">
             <number>8</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="documentNameLabel">
              <property name="styleSheet">
               <string notr="true">color: #1A1A1A;
font-family: Open Sans;
font-weight: 500;
font-size: 16px;
line-height: 22px;</string>
              </property>
              <property name="text">
               <string>Document Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="documentNameEdit">
              <property name="styleSheet">
               <string notr="true">QLineEdit {
	border: 1px solid #CCCCCC;
	border-radius: 8px;
	padding: 10px 16px;
	font-family: Open Sans;
	font-size: 14px;
	color: #1A1A1A;
	background-color: white;
}
QLineEdit:focus {
	border: 1px solid #FCCA69;
}
QLineEdit::placeholder {
	color: #999999;
}</string>
              </property>
              <property name="placeholderText">
               <string>Document name here</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="purposeWidget" native="true">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QVBoxLayout" name="purposeLayout">
            <property name="spacing">
             <number>8</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="purposeLabel">
              <property name="styleSheet">
               <string notr="true">color: #1A1A1A;
font-family: Open Sans;
font-weight: 500;
font-size: 16px;
line-height: 22px;</string>
              </property>
              <property name="text">
               <string>Purpose</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="purposeEdit">
              <property name="styleSheet">
               <string notr="true">QLineEdit {
	border: 1px solid #CCCCCC;
	border-radius: 8px;
	padding: 10px 16px;
	font-family: Open Sans;
	font-size: 14px;
	color: #1A1A1A;
	background-color: white;
}
QLineEdit:focus {
	border: 1px solid #FCCA69;
}
QLineEdit::placeholder {
	color: #999999;
}</string>
              </property>
              <property name="placeholderText">
               <string>Purpose here</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="descriptionWidget" native="true">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QVBoxLayout" name="descriptionLayout">
            <property name="spacing">
             <number>8</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="descriptionLabel">
              <property name="styleSheet">
               <string notr="true">color: #1A1A1A;
font-family: Open Sans;
font-weight: 500;
font-size: 16px;
line-height: 12px;</string>
              </property>
              <property name="text">
               <string>Description</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="descriptionEdit">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>100</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QTextEdit {
	border: 1px solid #CCCCCC;
	border-radius: 8px;
	padding: 10px 16px;
	font-family: Open Sans;
	font-size: 14px;
	color: #1A1A1A;
	background-color: white;
}
QTextEdit:focus {
	border: 1px solid #FCCA69;
}</string>
              </property>
              <property name="placeholderText">
               <string>Description here</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="bottomGroup" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>42</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>42</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>16</number>
         </property>
         <property name="leftMargin">
          <number>24</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>24</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>42</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	background-color: white;
	color: #1A1A1A;
	font-family: Open Sans;
	font-weight: 500;
	font-size: 18px;
	line-height: 24px;
	letter-spacing: 0%;
	border: 2px solid #1A1A1A;
	border-radius: 8px;
}
QPushButton:hover {
	background-color: #F5F5F5;
}</string>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>42</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	background-color: #FCCA69;
	color: white;
	font-family: Open Sans;
	font-weight: 500;
	font-size: 18px;
	line-height: 24px;
	letter-spacing: 0%;
	border-radius: 8px;
}
QPushButton:hover {
	background-color: #E6B85A;
}</string>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../Resources/Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>AddDocumentDialog</receiver>
   <slot>onCloseButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>AddDocumentDialog</receiver>
   <slot>onCancelButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>AddDocumentDialog</receiver>
   <slot>onSaveButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSaveButtonClicked()</slot>
  <slot>onCancelButtonClicked()</slot>
  <slot>onCloseButtonClicked()</slot>
 </slots>
</ui>
